development/openstack-base-focal-wallaby/bundle.yaml

```diff
@@ -1,9 +1,9 @@
----
+# Please refer to the OpenStack Charms Deployment Guide for more information.
+# https://docs.openstack.org/project-deploy-guide/charm-deployment-guide
+
+# Next line used only by Ubuntu OpenStack testing
 local_overlay_enabled: true
```

Comment:
It would actually make more sense to remove this key entirely. As I said, setting it to true is superfluous and just creates confusion.